src/langcheck/metrics/_pairwise_text_quality_utils.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import annotations
+
+from typing import Callable
+
+
+class PairwiseComparisonPromptGenerator:
+
+    def __init__(self, base_prompt_template: Callable,
```

Comment:
It would be better if we can show the right arguments & return type of these template functions by either docstring or type annotations.

Reply:
added both type hints and a docstring!